storage/src/vespa/storage/distributor/distributor_bucket_space.h

```diff
@@ -24,17 +24,17 @@ namespace distributor {
  *   particular so that redundancy, ready copies etc can differ across
  *   bucket spaces.
  */
-class ManagedBucketSpace {
+class DistributorBucketSpace {
```

Comment:
Consider updating the class comment from "A managed bucket space (...)" -> "A distributor bucket space (...)"